sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -357,7 +355,8 @@ case class FileSourceScanExec(
     val bucketed =
       selectedPartitions.flatMap { p =>
         p.files.map { f =>
-          val hosts = getBlockHosts(getBlockLocations(f), 0, f.getLen)
+          val hosts = DefaultFilePartitionStrategy.getBlockHosts(
+            DefaultFilePartitionStrategy.getBlockLocations(f), 0, f.getLen)
```

Comment:
How about using the object name `FilePartitionStrategy` instead of `DefaultFilePartitionStrategy` if these methods are general across all partitioning strategy?